modules/cactus-web/src/Button/Button.test.tsx

```diff
@@ -1,19 +1,85 @@
 import * as React from 'react'
 import { cleanup, render, fireEvent } from 'react-testing-library'
 import Button from './Button'
-import { ThemeProvider } from 'styled-components'
 import cactusTheme from '@repay/cactus-theme'
+import { ThemeProvider } from 'styled-components'
 
 afterEach(cleanup)
 
 describe('component: Button', () => {
-  test('snapshot', () => {
-    const { container } = render(
+  test('should default to standard variant', () => {
+    const button = render(
+      <ThemeProvider theme={cactusTheme}>
+        <Button>
+          Click me!
+        </Button>
+      </ThemeProvider>
+    )
+
+    expect(button).toMatchSnapshot()
```

Comment:
We'll need to snapshot just the rendered component instead of the whole wrapper
```suggestion
    expect(button.asFragment()).toMatchSnapshot()
```

Reply:
Ah, right! I'll update the tests